tests/search/bugs/4711376/app/services.xml

```diff
@@ -10,7 +10,8 @@
 
   <container version="1.0">
     <nodes>
-      <node hostalias="node1" jvmargs="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005"/>
+      <jvm options="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005" />
```

Comment:
Do we actually need debug options in a system test ?
I suspect that is options used for debugging locally that got committed by accident.

Reply:
Yeah, fixed